### DISCUSSION TITLE
iVentoy easy pxe ISO Boot

### DISCUSSION THREAD

**Discussion by snickerweb** [tteck/Proxmox#1540] [Ideas]
Hi. My idea for an installation script would be iVentoy. A simple pxe server. eg for ISO installations without USB stick. I have tested it. It works in the Debian12 LXC privileged.

You just have to start the server with bash iventoy.sh -A start

iVentoy Homepage: https://www.iventoy.com/en/index.html
Forum Thread about boot iventoy in lxc: https://forums.ventoy.net/showthread.php?tid=2491

**Comment by anvilzero** [comment]
What would be a good way to link Proxmox ISO Images to the ISO folder of iVentory? I just installed iVentoy on my Proxmox node directly and then linked the folders/ISOs. iVentoy as an LXC container would be great, especially if it could easily mirror what's already on Proxmox. Then you can just upload via the web UI.

`ln -s /var/lib/vz/template/iso/nixos-gnome-23.05-x86_64-linux.iso /root/iventoy-1.0.xx/iso/nixos-gnome-23.05-x86_64.iso`

**Comment by midzelis** [comment]
Hmm, I'm familiar with ventoy, but haven't tried netboot yet - that might be a nice way boot https://github.com/midzelis/zquickinit - which is my take on proxmox installer script (which supports ZFS encrypted roots, as well as ZFSBootmenu)
